certbot-apache/certbot_apache/constants.py

```diff
@@ -62,9 +62,9 @@
     vhost_files="*.conf",
     logs_root="/var/log/apache2",
     version_cmd=['/usr/sbin/apache2', '-v'],
-    define_cmd=['apache2ctl', 'virtualhosts'],
-    restart_cmd=['apache2ctl', 'graceful'],
-    conftest_cmd=['apache2ctl', 'configtest'],
+    define_cmd=['/usr/sbin/apache2', '-S'],
```

Comment:
I think this breaks the work @joohoi did on #2701. On a Gentoo system with OpenRC, the output from each command is:
```
# apache2ctl virtualhosts
 * No permission to apply cgroup settings
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
VirtualHost configuration:
*:443                  localhost (/etc/apache2/vhosts.d/00_default_ssl_vhost.conf:11)
*:80                   localhost (/etc/apache2/vhosts.d/00_default_vhost.conf:35)
ServerRoot: "/usr/lib64/apache2"
Main DocumentRoot: "/usr/htdocs"
Main ErrorLog: "/var/log/apache2/error_log"
Mutex ssl-cache: dir="/run/apache_ssl_mutex" mechanism=fcntl 
Mutex default: dir="/var/run/" mechanism=default 
Mutex mpm-accept: using_defaults
Mutex rewrite-map: using_defaults
Mutex ssl-stapling-refresh: using_defaults
Mutex ssl-stapling: using_defaults
PidFile: "/run/apache2.pid"
Define: DEFAULT_VHOST
Define: INFO
Define: SSL
Define: SSL_DEFAULT_VHOST
Define: LANGUAGE
Define: DUMP_VHOSTS
Define: DUMP_RUN_CFG
User: name="apache" id=81
Group: name="apache" id=81
```
```
# apache2 -S
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
VirtualHost configuration:
ServerRoot: "/usr/lib64/apache2"
Main DocumentRoot: "/usr/htdocs"
Main ErrorLog: "/var/log/apache2/error_log"
Mutex rewrite-map: using_defaults
Mutex default: dir="/var/run/" mechanism=default 
Mutex mpm-accept: using_defaults
PidFile: "/run/apache2.pid"
Define: DUMP_VHOSTS
Define: DUMP_RUN_CFG
User: name="apache" id=81
Group: name="apache" id=81
```
Notice the missing `Define` output.